package.json

```diff
@@ -93,12 +101,12 @@
     "output-file-sync": "^1.1.1",
     "phantomjs-prebuilt": "^2.1.7",
     "portfinder": "^1.0.3",
-    "react": "^15.0.0",
-    "react-component-metadata": "^2.1.1",
-    "react-dom": "^15.0.0",
+    "react": "^15.0.1",
+    "react-component-metadata": "^3.0.0",
```

Comment:
@jquense seems like `react-component-metadata 3.0.0` is incompatible with the react-bootstrap components when they are transpiled with Babel 6. Any chance you could give some pointers on how to make an isolated test case for this?